data/Hong_Kong/Legislative_Council/sources/instructions.json

```diff
@@ -6,6 +6,21 @@
       "type": "membership"
     },
     {
+      "file": "morph/term-6.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/hong_kong_legislative_council_members",
+        "query": "SELECT *, REPLACE(id,'yr16-20/','') AS id FROM data ORDER BY id"
+      },
+      "source": "http://www.legco.gov.hk/",
+      "type": "membership",
+      "merge": {
+        "incoming_field": "name",
+        "existing_field": "name",
+        "reconciliation_file": "reconciliation/term-6.csv"
+      }
```

Comment:
The IDs look as if they're consistent between terms, which means you shouldn't need this merge block (or the reconciliation file).